example/integration_tests/components-ce/.bazelrc

```diff
@@ -1 +1,3 @@
 build --enable_bzlmod
+build --java_runtime_version=graalvm_20
+build --tool_java_runtime_version=graalvm_20
```

Comment:
this is how you use the GraalVM JVM as your Java Toolchain in Bazel, which you must do so that it can find the GraalVM `org.graalvm.polyglot.Context` type.

the other way to do this is to install and build against `org.graalvm.sdk:graal-sdk` from Maven, which provides these symbols on non-GraalVM runtimes.